synapse/types.py

```diff
@@ -691,6 +691,7 @@ class StreamToken:
     push_rules_key: int
     to_device_key: int
     device_list_key: int
+    # Note that the groups key is no longer used and may have bogus values.
     groups_key: int
```

Comment:
Or maybe we should always set this to 0?

```suggestion
    # Note that the groups key is no longer used and may have bogus values.
    _groups_key: int = 0
```

Reply:
I don't think we should default it, as I think that might confuse things greatly if we add another stream at a later point?

Reply:
That was my thought too. 👍